src/auto_intersphinx/catalog.py

```diff
@@ -101,7 +101,10 @@ def docurls_from_environment(package: str) -> dict[str, str]:
             return {}
         for k in md.get_all("Project-URL"):
             if k.startswith(("documentation, ", "Documentation, ")):
-                addr = _ensure_webdir(k.split(",", 1)[1].strip())
+                raw_addr = k.split(",", 1)[1].strip()
+                # Find the final address after eventual redirects.
+                redirected = requests.head(raw_addr, allow_redirects=True).url
+                addr = _ensure_webdir(redirected)
```

Comment:
@anjos maybe you want to review changes done to the source code and tests

Reply:
Yes, I'd appreciate it if you could give it a quick look.
Basically:
- sphinx 7.4.0 changed their output, so I changed the tests to accept either output;
- some tests failed when looking at readthedocs URLs because `objects.inv` was not at the URL specified in the project's metadata, so I made sure to use a URL that should contain `objects.inv` (following redirects).

Reply:
Ah, I see he is unavailable for now.
I'll merge this.